db/re/quest_db.yml

```diff
@@ -5560,9 +5598,40 @@ Body:
   - Id: 8583
     Title: SACOM's performance analysis - Standby
     TimeLimit: 4h
+  - Id: 8586
+    Title: Keeping Breeding Ground
   - Id: 8587
     Title: Daily farm experience - Standby
     TimeLimit: 4h
+  - Id: 8588
+    Title: Tail Collection
+  - Id: 8589
+    Title: It's Boss Meow!
```

Comment:
Title: "It's Boss Meow!"

Reply:
(feel free to use the suggestion feature for small review)
![Capture d’écran (1115)](https://user-images.githubusercontent.com/5231893/201674365-55d27f1d-2280-4f15-b3e6-55bbdcbd057d.png)